doc/tutorial/Authentication.lhs

```diff
@@ -281,17 +281,22 @@ lookupAccount key = case Map.lookup key database of
 For generalized authentication, servant exposes the `AuthHandler` type,
 which is used to wrap the `Request -> Handler user` logic. Let's
 create a value of type `AuthHandler Request Account` using the above `lookupAccount`
-method:
+method (note: we depend upon `cookie`'s `parseCookies` for this):
```

Comment:
I know this is going to sound "annoying", but could you please have "`cookie`" point to the package's page on hackage? To make it clear and unambiguous what "`cookies` refers to here.